src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2509,14 +2509,14 @@
 type: command
 short-summary: Place the CLI in a waiting state until a condition of a scale set is met.
 examples:
-  - name: Place the CLI in a waiting state until a condition of a scale set is met. (autogenerated)
-    text: |
-        az vmss wait --created --name MyScaleSet --resource-group MyResourceGroup
-    crafted: true
   - name: Place the CLI in a waiting state until the VMSS has been updated.
     text: az vmss wait --updated --name MyScaleSet --resource-group MyResourceGroup
   - name: Place the CLI in a waiting state until the VMSS instance has been updated.
     text: az vmss wait --updated --instance-id 1 --name MyScaleSet --resource-group MyResourceGroup
+  - name: Place the CLI in a waiting state until a condition of a scale set is met. (autogenerated)
+    text: |
+        az vmss wait --created --name MyScaleSet --resource-group MyResourceGroup
+    crafted: true
```

Comment:
Why this one is moved down? Is there some weight calculation for the order?

Reply:
My problem think human written example has higher priority. So I put it ahead of Aladdin generated.